shapely/strtree.py

```diff
@@ -94,8 +105,8 @@ def geometries(self):
     def query(self, geometry, predicate=None, distance=None):
         """
         Return the integer indices of all combinations of each input geometry
-        and tree geometries where the extent of each input geometry intersects
-        the extent of a tree geometry.
+        and tree geometries where the two-dimensional axis-aligned bounding box
+        of each input geometry intersects the bounding box of a tree geometry.
```

Comment:
Is the "axis-aligned" needed here? Or can we understand "bounding box" to always be axis aligned? 

The reason I am asking is that it, while maybe strictly more correct, also makes this summary paragraph more complex to read. 

Wondering if we could leave the "two dimensional" and "axis-aligned" for a second (or other) paragraph where details are given what kind of bounding box is used and that z dimensions are ignored, and keeping the first summary paragraph simpler.